egs2/TEMPLATE/asr2/asr2.sh

```diff
@@ -431,8 +433,13 @@ if [ ${kmeans_feature} = "mfcc" ]; then  # MFCC has no layer
 else
     kmeans_feature_type=$(echo "${kmeans_feature}" | cut -d/ -f1)
     layer=$(echo "${kmeans_feature}" | cut -d/ -f2)
-    # TODO(simpleoier): to support features beyond s3prl
-    s3prl_conf="{upstream=${kmeans_feature_type}}"
+
+    if [ $upstream == "s3prl" ]; then
+        s3prl_conf="{upstream=${kmeans_feature_type}}"
```

Comment:
Thanks! How about renaming `s3prl_conf` as `upstream_conf` (I randomly came up with this name. feel free to propose other names)?

Reply:
Of course! I will change it to `upstream_conf`.